policies.go

```diff
@@ -970,7 +979,8 @@ func (d *rackAwareRR) AddHost(host *HostInfo) {
 
 func (d *rackAwareRR) RemoveHost(host *HostInfo) {
 	dist := d.HostTier(host)
-	d.hosts[dist].remove(host.ConnectAddress())
+	connAddr, _ := host.ConnectAddress()
+	d.hosts[dist].remove(connAddr)
```

Comment:
same as above, check in `AddHost` so we don't add hosts with invalid addresses and then we can just return here when unable to get connAddr